lib/stripe/util.rb

```diff
@@ -91,47 +91,50 @@ def self.convert_to_stripe_object(data, opts = {})
 
     def self.log_error(message, data = {})
       if !Stripe.logger.nil? ||
-          !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_ERROR
+         !Stripe.log_level.nil? && Stripe.log_level <= Stripe::LEVEL_ERROR
         log_internal(message, data, color: :cyan,
-          level: Stripe::LEVEL_ERROR, logger: Stripe.logger, out: $stderr)
+                                    level: Stripe::LEVEL_ERROR, logger: Stripe.logger, out: $stderr)
```

Comment:
Also not a huge fan of this indent style. I'd prefer using either a 2-space indent or align the second line with the first parameter, not with the first _keyword_ parameter.

Reply:
+1. For the same reason as above, it's just impractical in that it doesn't do anything to improve readability (over two-space style), but creates unnecessarily large diffs anytime the method or any of these variables changes name.

Reply:
Okay, so the problem with this one is that named parameters implicitly form a hash, so they're handled by hash alignment rules. AFAICT, it's possible to tell Rubocop to ignore or inspect hashes as arguments (implicit or explicit) but not to apply different alignment rules than for "normal" hashes.

A simple fix is to start named parameters on a newline, but it's a shame that Rubocop can't handle that automatically for us.

Reply:
Ah, cool. Thanks for investigating!

Honestly in this case, `color` would probably be better off started on a newline anyway.